<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="btnOpenCamera">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>300</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCapture">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>300</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>捕获图片</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCamera">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>摄像头</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCapture">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>捕获图</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResult">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>结果图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGray">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>300</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>灰度化</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReadImage">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>300</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>打开图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnThreshold">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>300</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>阈值分割(Otsu)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCapture</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>btnCapture_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOpenCamera</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>btnOpenCamera_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnReadImage</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>btnReadImage_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnGray</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>btnGray_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnThreshold</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>btnThreshold_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnOpenCamera_Clicked()</slot>
  <slot>btnCapture_Clicked()</slot>
  <slot>btnReadImage_Clicked()</slot>
  <slot>btnGray_Clicked()</slot>
  <slot>btnThreshold_Clicked()</slot>
 </slots>
</ui>
